insights/parsers/tests/test_yum_repos_d.py

```diff
@@ -18,6 +19,12 @@
 gpgcheck=1
     0 # This should be ignored
 gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-beta,file:///etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release
+
+[bad-unicode]
+baseurl = https://example.com
+name = insights-test-💩
```

Comment:
I am afraid this test can’t actually work, because you are reading a Unicode string here, not a set of bytes.